tests/wpt/tests/css/CSS2/stacking-context/root-element-creates-stacking-context.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html style="width: 0; height: 0; border: 50px solid red">
+  <link rel="help" href="https://drafts.csswg.org/css2/#stacking-context">
+  <link rel="match" href="root-element-creates-stacking-context-ref.html">
+  <link rel="author" title="Michael Rees" href="mailto:mrees@noeontheend.com">
+  <meta name="" content="root element forms the root stacking context">
+  <body style="border: 50px solid green; margin: 0; position: relative; inset: -50px; z-index: -1">
```

Comment:
`inset` is not part of CSS2, margin should do the trick too:

```suggestion
  <body style="border: 50px solid green; position: relative; margin: -50px; z-index: -1">
```